client/tests/acceptance/user-can-edit-deis-test.js

```diff
@@ -0,0 +1,84 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  click,
+  findAll,
+  currentURL,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+
+module('Acceptance | user can edit DEIS Packages', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(() => {
+    authenticateSession({
+      emailaddress1: 'me@me.com',
+    });
+  });
+
+  test('User sees Project under Awaiting Submission if it has an active Draft EIS', async function (assert) {
+    this.server.create('project', {
+      packages: [
+        this.server.create('package', 'toDo', 'deis'),
+        this.server.create('package', 'done', 'deis'),
+      ],
+    });
+
+    this.server.create('project', {
+      packages: [
+        this.server.create('package', 'done', 'deis'),
+      ],
+    });
+
+    await visit('/projects');
+    assert.equal(findAll('[data-test-projects-list="to-do"] [data-test-my-project-list-item]').length, 1);
+
+    assert.equal(findAll('[data-test-projects-list="done"] [data-test-my-project-list-item]').length, 1);
```

Comment:
Just a comment, but it's sometimes a stronger assertion to look for some specific text, perhaps a fake project name.